### PR TITLE
Cal 15

### PR DESCRIPTION
**Pull request by xorver**

### CODE REVIEW

**Comment by xorver** [comment]
This pull request has spring security config that permits everything, I'll add correct configuration in separate branch

File: src/main/webapp/resources/fonts/glyphicons-halflings-regular.svg [hunk 1]
```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (pkrp) [inline src/main/webapp/resources/fonts/glyphicons-halflings-regular.svg]: You don't need to add all font-awesome files to repo. Just add proper line to dependencies in bower.json file (e.g. "fontawesome": "4.3.0" - don't know which version exactly you have used). Then execute 'bower install' and add proper paths to index.html.

File: src/main/webapp/bower.json [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
+    ],
+    "dependencies": {
+        "angularjs": "1.3.15",
+        "angular-route": "1.3.15",
+        "bootstrap": "3.3.4",
+        "angular-ui-map": "0.5.0",
+        "angular-ui-calendar": "0.8.1",
+        "angular-bootstrap": "0.12.1",
+        "restangular": "1.5.1",
+        "lodash": "3.8.0"
```
Comment (pkrp) [inline src/main/webapp/bower.json]: You don't need to add all that font-awesome files to repo. Just add proper line to dependencies here (e.g.  "fontawesome": "4.3.0", don't know which exactly version you have used) and add paths to proper files in index.html

Comment (xorver) [inline src/main/webapp/bower.json]: done

**Pull request merged**